### PR TITLE
chore: migrates rss to v2

### PR DESCRIPTION
**Pull request by dzucconi**
Ports `/rss/partner-updates` to TypeScript. Drops `/rss/news` since we no longer publish news.

### CODE REVIEW

File: src/v2/Apps/RSS/rssServerApp.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+rssServerApp.get("/rss/partner-updates", async (req, res) => {
+  const relayEnvironment = createRelaySSREnvironment({
+    userAgent: req.header("User-Agent"),
+  })
+
+  const { articlesConnection } = await fetchQuery<RssPartnerUpdatesQuery>(
+    relayEnvironment,
+    RSS_PARTNER_UPDATES_QUERY,
+    { channelId: GALLERY_PARTNER_UPDATES_CHANNEL }
+  )
```
Comment (dzucconi) [inline src/v2/Apps/RSS/rssServerApp.tsx:24]: I believe this is the first instance of us just using Relay to fetch server-side without React? So no need for that `metaphysics2` function.

Comment (dzucconi) [reply src/v2/Apps/RSS/rssServerApp.tsx:24]: Could also just be a normal `fetch`, I suppose.

File: src/v2/Apps/RSS/templates/article.ejs [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<item>
+  <guid isPermaLink="false"><%= appUrl %><%= article.href %></guid>
+  <title><%= article.thumbnailTitle %></title>
+  <link><%= appUrl %><%= article.href %></link>
+  <author><%= article.byline %></author>
```
Comment (dzucconi) [inline src/v2/Apps/RSS/templates/article.ejs:5]: This is technically Invalid FWIW. This is the old behavior but according to spec `author` should be the author's email address. I'm assuming other RSS feeds abuse this field too.

File: src/v2/Apps/RSS/rssServerApp.tsx (hunk 1 above)
Comment (damassi) [reply src/v2/Apps/RSS/rssServerApp.tsx:24]: generated relay types are always good

**Pull request merged**